### PR TITLE
test: Test resource view

### PR DESCRIPTION
**Pull request by yevgenypats**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

This is blocked by  https://github.com/cloudquery/cq-provider-sdk/pull/305 

This will help find issues in our CI early on and keep the aggregated view in sync with our schema.

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [ ] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [ ] If adding a new resource, add [relevant Terraform files](../terraform) in a separate PR 📂
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -41,4 +41,4 @@ __debug_bin
 terraform/downloaded*
 config.hcl
 terraform/tfplan.binary
-cq-provider
+cq-provider-gcp
```
Comment (erezrokah) [inline .gitignore:44]:
```suggestion
cq-provider
```

Is this change needed? `make run` creates a `cq-provider` binary

**Review by erezrokah — APPROVED** [review]
Looks great, pending one comment

File: .gitignore (hunk 1 above)
Comment (yevgenypats) [reply .gitignore:44]: yeah but go build creates `cq-provider-gcp` maybe we need both ?

Comment (yevgenypats) [reply .gitignore:44]: added both.
